### PR TITLE
[4.0] Remove tabindex

### PR DESCRIPTION
**Pull request by brianteeman**
There is no need to define a tabindex when the order is the natural order

Code review

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 3a895242f09a14ee15adfbde5dd9fd74b1461709<br /><br />Code Review - both. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28192">issues.joomla.org/tracker/joomla-cms/28192</a>.</sub>

File: administrator/components/com_joomlaupdate/tmpl/update/finaliseconfirm.php [hunk 1]
```diff
@@ -85,10 +85,10 @@
 		<div class="control-group">
 			<div class="controls">
 				<div class="btn-group">
-					<a tabindex="4" class="btn btn-danger" href="index.php?option=com_joomlaupdate">
+					<a class="btn btn-danger" href="index.php?option=com_joomlaupdate">
 						<span class="fas fa-times" aria-hidden="true"></span> <?php echo Text::_('JCANCEL'); ?>
 					</a>
-					<button tabindex="5" class="btn btn-primary">
+					<button type="button" class="btn btn-primary">
```
Comment (Quy) [inline administrator/components/com_joomlaupdate/tmpl/update/finaliseconfirm.php]:
```suggestion
					<button type="submit" class="btn btn-primary">
```

File: administrator/components/com_joomlaupdate/tmpl/upload/captive.php [hunk 2]
```diff
@@ -85,10 +85,10 @@
 		<div class="control-group">
 			<div class="controls">
 				<div class="btn-group">
-					<a tabindex="4" class="btn btn-danger" href="index.php?option=com_joomlaupdate">
+					<a class="btn btn-danger" href="index.php?option=com_joomlaupdate">
 						<span class="fas fa-times icon-white" aria-hidden="true"></span> <?php echo Text::_('JCANCEL'); ?>
 					</a>
-					<button tabindex="5" class="btn btn-primary">
+					<button type="button" class="btn btn-primary">
```
Comment (Quy) [inline administrator/components/com_joomlaupdate/tmpl/upload/captive.php]:
```suggestion
					<button type="submit" class="btn btn-primary">
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on e7c406313babe415d45cbc8b4b0d2a0b78ba7fac<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28192">issues.joomla.org/tracker/joomla-cms/28192</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28192">issues.joomla.org/tracker/joomla-cms/28192</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**